openedx/core/djangoapps/notifications/permissions.py

```diff
@@ -0,0 +1,27 @@
+"""
+Permissions for notifications
+"""
+from edx_rest_framework_extensions.auth.jwt.authentication import JwtAuthentication
+from edx_rest_framework_extensions.auth.session.authentication import SessionAuthenticationAllowInactiveUser
+from rest_framework.permissions import IsAuthenticated
+
+from openedx.core.lib.api.authentication import BearerAuthenticationAllowInactiveUser
+
+
+def view_auth_classes():
```

Comment:
The name is not descriptive enough, it adds a permission class not auth class,  it should be something like , `allow_authenticated_but_unverified_permissions`

Reply:
It is adding `permission_class` and `authentication_class`. I used the same name which is being used in discussion.

Reply:
Sorry I missed that, But still, the name is not descriptive.

Reply:
Updated decorator name